library/extractor/build.gradle

```diff
@@ -19,7 +19,11 @@ android {
             testCoverageEnabled = true
         }
     }
-
+    testOptions{
+        unitTests.all {
+            jvmArgs '-noverify'
```

Comment:
This allows you to run Robolectric tests with coverage from the Studio UI.  I find it very useful, but it is definitely optional.  FYI: You used to be able to specify -noverify in the UI, but that seems to be broken in the latest version of Studio.

Reply:
Thanks for the clarification. Please kindly fresh a new issue or pull request for this specifically. I would like to ask the Android studio team about this. If it's just an Android Studio bug, it might make sense not to add it in case it has other unintended effects.

Reply:
I've used Robolectric for a number of years and the verifier has always been an issue with IntelliJ code coverage.  Jacoco doesn't have an the same issue, but it has it's own set of problems with Robolectric.  We used IntelliJ in Studio and Jacoco for CI, so I have some battle scars!

Reply:
Done